lib/buildpackutil.py

```diff
@@ -29,6 +29,14 @@ def get_vcap_data():
             "application_name": "My App",
         }
 
+#Added by Arif for App Insights
+def get_container_name():
+    logging.debug("Getting container name.")
+    return (os.environ["APPNAMESPACE"]+"-Leader" if os.getenv("CF_INSTANCE_INDEX", "")=="0" else os.environ["APPNAMESPACE"]+"-Slave")
```

Comment:
the leader and slave are lower case. 
eg. coo-acd-dsradevopsdemo-mdx-ui-qa02-leader, coo-acd-dsradevopsdemo-mdx-ui-qa02-slave